--- polyfill.js	2021-09-24 09:18:20.919285676 +0200
+++ polyfill.js	2021-09-24 09:20:16.311451537 +0200
@@ -310,6 +310,11 @@
 // Use polyfill for setImmediate for performance gains
 Promise._immediateFn =
   // @ts-ignore
+  (typeof queueMicrotask == 'function' &&
+    function (fn) {
+      // @ts-ignore
+      queueMicrotask(fn);
+    }) ||
   (typeof setImmediate === 'function' &&
     function(fn) {
       // @ts-ignore
